auto_round/mllm/autoround_mllm.py

```diff
@@ -155,14 +159,15 @@ def __init__(
         if nsamples % batch_size != 0:
             nsamples = (nsamples // batch_size + 1) * batch_size
             logger.warning(f"'nsamples' is not divisible by 'batch_size', will adjusted to {nsamples}")
-
-        if quant_nontext_module or (dataset in CALIB_DATASETS.keys() and not _only_text_test(model, tokenizer)):
+            
+        if not isinstance(dataset, torch.utils.data.DataLoader) and (
+                quant_nontext_module or (dataset in CALIB_DATASETS.keys() and not _only_text_test(model, tokenizer))):
             if quant_nontext_module:
                 logger.warning(f"Quantitative nontext module is not supported for plain text datasets,"
```

Comment:
Plain text datasets are unsuitable for calibrating non-text modules, switching to liuhaotian/llava_conv_58k